views/forms/TextInputTopLabel/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { OutlinedInput, InputLabel } from '@mui/material/'
+import { useController } from 'react-hook-form'
+
+const TextInputTopLabel = (props) => {
+  const { errors } = props
```

Comment:
you should be able to get errors from the field

Reply:
0730f7e